README.md

```diff
@@ -19,6 +19,11 @@ You can then run Azure CLI scripts to create and manage any Azure resource.
 
 Refer to [starter templates](https://github.com/Azure/actions-workflow-samples/tree/master/AzureCLI) for examples.
 
+## Manage resources by deploying ARM Templates
+-  [ARM](https://github.com/Azure/ARM)(`azure/arm`) action can be used to specify a resource group, location and deploy the ARM templates for creating, updating and deleting resources. 

```

Comment:
Shouldn't the action be called something like `arm-template-deploy`?

Reply:
Also,  is this repo private?  You may want to make it public and publish it to the marketplace before adding it here.

Reply:
> Shouldn't the action be called something like `arm-template-deploy`?

Point noted. Will make relevant changes.